### DISCUSSION TITLE
مشکل دریافت نکردن اپدیت new_chat_memebrs

### DISCUSSION THREAD

**Discussion by aaq1386** [kianahmadian/python-bale-bot#155] [Q&A] [answered]
با سلام 
آپدیت های عضویت در گروه به سیستم ارسال نمیشه و ربات تشخیص نمیده فقط اپدیت خروج مشخص میشه
```
@bot.event
async def on_message(ms: Message):
    print(ms)  

    if ms.new_chat_members: 
        await ms.reply(text=f"🎉 خوش آمدی  به گروه! 🎉")
     ```

**Comment by kian-ahmadian** [comment]
درود بر شما عزیز 
این مشکل احتمالاً مربوط به بله‌ست. تنظیمات و دسترسی های بازو در گروه و یا کانال مورد نظرتون و در بالاترین سطح قرار بدید و سپس دوباره امتحان کنید.

در صورتی که تغییر حاصل نشد، بفرمایید تا بررسی کنیم.

**Reply by aaq1386** [reply]
بعد از برسی دسترسی های ربات و تست مجدد به نتیجه زیر رسیدم:
با متد left_chat_member دستور را اجرا شده  و اپدیت را دریافت و پیام مورد نظر را ارسال کرد ولی متد new_chat_members اپدیتی ارسال نمیکند
و پاسخی ندارد!
با تشکر

**Reply by aaq1386** [reply]
تنظیمات بییشترین حد هست
حتی چند بار ربات را تعویض کردم

**Comment by kian-ahmadian** [comment]
در ضمن کدی که ظاهراً برای ریپلای کردن بر روی پیام ورود نوشته اید، اشتباه هست. 
مشخصه `new_chat_members`  که در کلاس `bale.Message` قرار دارد یک لیستی از شئ `bale.User` رو برمیگردونه.
توصیه میکنم حتما به داکیومنت سری بزنید و همچنین نمونه کد های قبل تر (مربوط به نسخه استیبل ۲.۵.۰) رو بررسی کنید.
موفق باشید

**Comment by kian-ahmadian** [comment, marked as answer]
اصلاح پاسخ: این مشکل از طرف بله رُخ می‌داد و دو روز پس از آن‌که به مسئولین مربوطه ارجاع داده‌شد، برطرف گشت.
از گزارش شما بسیار سپاسگزارم
